mathics/eval/pymathics.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
```

Comment:
I had a comment about this in the other PR that I guess comes after this. Please see that.